crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -331,6 +331,7 @@ en:
         unknown: '%{count} Unknown'
         unready: '%{count} Not Ready'
         pending: '%{count} Pending'
+        crowbar_upgrade: '%{count} Upgrade'
```

Comment:
Question: Wont you need to add "Upgrade" in # Global state list too in this file ? (around Line 627)